evap/staff/templates/staff_questionnaire_form.html

```diff
@@ -58,9 +58,19 @@ <h5 class="card-title">{% translate 'Questions' %}</h5>
                                         <div class="form-check mt-1">
                                             <input class="form-check-input allow-textanswer-checkbox" id="{{ form_element.allows_additional_textanswers.id_for_label }}" name="{{ form_element.allows_additional_textanswers.html_name }}"
                                                 type="checkbox" data-keep="true" {% if form_element.allows_additional_textanswers.value %} checked{% endif %}
-                                                {% if form_element.allows_additional_textanswers.field.disabled %} disabled{% endif %} />
+                                                {% for attr_name, attr_value in form_element.allows_additional_textanswers.field.widget.attrs.items %}
+                                                    {{ attr_name }}="{{ attr_value }}"
+                                                {% endfor %} />
```

Comment:
My intention here was to `include` the template from django where this is implemented. If we can't do that, for some reason, let's stick to the explicitly rendered `disabled` attribute (like you had before).